larva/src/main/java/org/frankframework/larva/TestConfig.java

```diff
@@ -51,7 +51,9 @@ public void incrementScenarioCounter() {
 	public void flushWriters() {
 		try {
 			if (out != null) {
-				out.flush();
+				synchronized (out) {
```

Comment:
Is het haalbaar om synchronized te doen alleen tijdens het multi-threaded draaien van scenarios?
Anders geeft het onnodige overhead voor iedere single-threaded run.